### ISSUE TITLE
Question about resize function

### ISSUE THREAD

**Issue by seungjunlee96** [rosinality/stylegan2-pytorch#102]
Thank you for your amazing codes!

I have a question about before applying perceptual loss.

The code applied resize function as below
```python
            if height > 256:
                factor = height // 256

                img_gen = img_gen.reshape(
                    batch, channel, height // factor, factor, width // factor, factor
                )
                img_gen = img_gen.mean([3, 5])
```

However, I found `torch.nn.functional.interpolate` can easily resize the images, too.
`percept(F.interpolate(real, size = 256), F.interpolate(fake, size = 256)).mean()`
So I wondered if there is a difference between the two and what it is.
Thanks,

**Comment by rosinality** [comment]
Yes, it will be same with bilinear interpolation. I just took it from official codes.
